kiota_authentication_azure/_observability.py

```diff
@@ -0,0 +1,24 @@
+from kiota_http.middleware.parameters_name_decoding_handler import ParametersNameDecodingHandler
+from kiota_http.observability_options import ObservabilityOptions
+from opentelemetry import trace
+
+from ._version import VERSION
+
+tracer = trace.get_tracer(ObservabilityOptions.get_tracer_instrumentation_name(), VERSION)
+
+
+class Observability:
+    """Helper object to manage creation of spans."""
+
+    def create_parent_span_name(self, uri: str, method: str) -> str:
+        """Creates a parent span name for the given method."""
+        name_handler = ParametersNameDecodingHandler()
+        uri_template = name_handler.decode_uri_encoded_string(uri)
+        parent_span_name = f"{method} - {uri_template}"
```

Comment:
do not include the URI in ANY span name (please double check the others) as it maybe contain EUPI. The URI is only added as an attribute to the main span by http IF the includeEUPI option is set to true.

Reply:
I have removed the uri previously used here. Updated it.